runtime/sema/gen/testdata/comparable.golden.go

```diff
@@ -0,0 +1,35 @@
+// Code generated from testdata/comparable.cdc. DO NOT EDIT.
+/*
+ * Cadence - The resource-oriented smart contract programming language
+ *
+ * Copyright Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sema
+
+const TestTypeName = "Test"
+
+var TestType = &SimpleType{
+	Name:          TestTypeName,
+	QualifiedName: TestTypeName,
+	TypeID:        TestTypeName,
+	tag:           TestTypeTag,
+	IsResource:    false,
+	Storable:      false,
+	Equatable:     false,
+	Comparable:    true,
+	Exportable:    false,
+	Importable:    false,
+}
```

Comment:
Is this generated test type being asserted anywhere? (i.e: to see if the generated type has the 'Comparable' field set to `true`)

Reply:
The `.golden.go` files are actually the expectations for the respective `.cdc` test cases.

Reply:
ah ok, got it!